numpy/core/tests/test_errstate.py

```diff
@@ -1,12 +1,18 @@
-import platform
 import pytest
+import sysconfig
 
 import numpy as np
 from numpy.testing import assert_, assert_raises
 
+# The floating point emulation on ARM EABI systems lacking a hardware FPU is
+# known to be buggy. This is an attempt to identify these hosts. It may not
+# catch all possible cases, but it catches the known cases of gh-413 and
+# gh-15562.
+hosttype = sysconfig.get_config_var('HOST_GNU_TYPE')
+arm_softfloat = False if hosttype is None else hosttype.endswith('gnueabi')
 
 class TestErrstate:
-    @pytest.mark.skipif(platform.machine() == "armv5tel", reason="See gh-413.")
+    @pytest.mark.skipif(arm_softfloat, reason="See gh-413, gh-15562.")
```

Comment:
Maybe you can make the reason longer?

Reply:
> Maybe you can make the reason longer?

Based on your previous suggestion, I 'platform/cpu issue with FPU'. I can make it longer, of course, but looking at the other tests, brevity seems to be preferred.